docs/user/dashboard/dashboard.asciidoc

```diff
@@ -116,56 +158,99 @@ To compare the data in the panels, arrange the panels on the dashboard, or remov
 
 * To resize, click the resize control, then drag to the new dimensions.
 
+* To maximize the panel to fullscreen, open the panel menu, then click *More > Maximize panel*.
+
 * To delete, open the panel menu, then select *Delete from dashboard*.
 
 . To save your changes, click *Save* in the toolbar.
 
+[float]
+[[apply-design-options]]
+=== Apply design options
+
+Apply a set of design options to the entire dashboard.
+
+. From the toolbar, click *Edit > Options*.
+
+. Select the following options:
+
+* *Use margins between panels* &mdash; Specifies a margin of space between each panel.
+
+* *Show panel titles* &mdash; Specifies the appearance of titles in the header of each panel.
+
+* *Sync color pallettes across panels* &mdash; Specifies the color pallette that is applied to all panels.
```

Comment:
Since there is no color palette editor here, it would be more correct to say something like:

> *Sync color palettes across panels* &mdash; Specifies whether the color palette is applied to all panels.